datasets/huishoudelijkafval/huishoudelijkafval.json

```diff
@@ -299,6 +367,53 @@
             "relation": "bag:nummeraanduidingen",
             "uri": "https://bag.basisregistraties.overheid.nl/def/bag#identificatiecode",
             "description": "Identificatie nummeraanduiding"
+          },
+          "containerlocatieTypeNaam": {
+            "type": "string",
+            "provenance": "containerlocatie_type_naam",
+            "description": "Het type van de containerlocatie"
+          },
+          "containerlocatieIdNummer": {
+            "type": "string",
+            "provenance": "containerlocatie_id_number",
+            "description": "Serienummer dat door de fabrikant aan het object is gegeven."
+          },
+          "containerlocatieDatumWijziging": {
+            "type": "string",
+            "format": "date-time",
+            "provenance": "containerlocatie_datum_wijziging",
+            "description": "Datum waarop het object is gewijzigd."
+          },
+          "containerlocatieOpmerking": {
+            "type": "string",
+            "provenance": "containerlocatie_opmerking",
+            "description": "Opmerking over het object."
+          },
+          "containerlocatieEndOfLife": {
+            "type": "string",
+            "format": "date",
+            "provenance": "containerlocatie_end_of_life",
+            "description": "Datum waarop het object het einde van zijn levenscyclus heeft bereikt."
+          },
+          "containerlocatieEigenaarschap": {
+            "type": "string",
+            "provenance": "containerlocatie_eigenaarschap",
+            "description": "Soort eigenaarschap"
+          },
+          "containerlocatieEigenaarschapOpmerking": {
+            "type": "string",
+            "provenance": "containerlocatie_eigenaarschap_opmerking",
+            "description": "Beschrijving van het type eigenaarschap."
+          },
+          "containerlocatie_containerlocatie_type_artikelcode": {
```

Comment:
Moet denk ik zijn: `containerlocatieTypeArtikelcode`.

Reply:
deze is ook aangepast maar deze was dubbel ene was goed de andere niet.